docs/pages/modules/use-standalone-expo-module-in-your-project.mdx

```diff
@@ -65,37 +79,43 @@ Compile and run the app with the following command:
   ]}
 />
 
-You can now use the module inside your app. To test this, let's edit the **App.js** file in the app and render the text message from the `expo-settings` module:
+You can now use the module in your app. To test it, edit the **App.js** file in your app and render the text message from the `expo-settings` module:
 
 ```jsx App.js
-import * as Settings from 'expo-settings';
-import { StatusBar } from 'expo-status-bar';
 import React from 'react';
 import { Text, View } from 'react-native';
+import * as Settings from 'expo-settings';
 
 export default function App() {
   return (
     <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
       <Text>{Settings.hello()}</Text>
-      <StatusBar style="auto" />
     </View>
   );
 }
 ```
 
-After this configuration, you'll see a text that says "Hello world! 👋" in the app.
+After this configuration, the app displays the text "Hello world! 👋".
+
+</Step>
 
 ## Publish the module to npm
 
-You can publish the module on npm and then add it to your project by installing it as a dependency.
+Publish the module on npm, then install it as a dependency in your project.
 
-### 1. Initialize a new module
+<Step label="1">
 
-Start by creating a new module using `create-expo-module`. Pay attention to each prompt as you'll publish this library, and choose a unique name for your npm package.
+### Initialize a new module
+
+Start by creating a new module with `create-expo-module`. Follow the prompts carefully, as you will publish this library, and choose a unique name for your npm package.
 
 <Terminal cmd={['$ npx create-expo-module expo-settings']} />
 
-### 2. Run the example project
+</Step>
+
+<Step label="2">
+
+### Run the example project
 
 Run the example project to make sure everything is working. Then, start the TypeScript compiler to watch for changes and rebuild the JavaScript module.
```

Comment:
```suggestion
Run one of your apps to ensure everything works. Then, start the TypeScript compiler in the root of your project to watch for changes and rebuild the module's JavaScript:
```

Let's use the same phrase from the previous section here to use parallelism (which is helpful for end users when they are trying to skim through the information and implement it at the same time).
Also, this section previously didn't mention where to run this command. We should add that here.